test/components/alert-test.jsx

```diff
@@ -16,6 +16,22 @@ describe('AlertComponent', function() {
 
   });
 
+  it('should render an alert without a H4 if title is no passed in', function() {
```

Comment:
I'm guessing all the tests have been done this way, but it would be nice if we could use some `describe`s to give us context and make the test reporting nicer to read.  Not a blocker, just a thought.